src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/ScenarioTests.kt

```diff
@@ -71,18 +71,6 @@ class ScenarioTests : DatabaseTest()
         }
     }
 
-    @Test
-    fun `only touchstone-preparer can get scenario for in-preparation touchstone`()
```

Comment:
is this test obsolete for some reason?

Reply:
Oh sorry, I meant to leave a comment about this: It was a duplicate test, presumably from a merge. There's an identical test with a slightly different name elsewhere in that class.